showcase/src/app/demo-ui/pages/reactive-forms-with-ngx-form-errors/demo-reactive-forms-with-ngx-form-errors-page.component.scss

```diff
@@ -0,0 +1,90 @@
+$monitor-query: "screen and (min-width:1200px)";
```

Comment:
Could you please use variables from "stark-ui" instead ? 😊 
```css
@import "~@nationalbankbelgium/stark-ui/assets/styles/media-queries";
```

Then you can use the variables `$desktop-screen-query`, `$tablet-only-query` and `$mobile-only-query`.